### PR TITLE
Makes Khopesh historicaly accurate

### PR DESCRIPTION
**Pull request by GregTheEgg42069**
#### Summary
Balance "Swaps the khopeshes dmg values to reflect irl use, and change the description to reflect it"

#### Purpose of change

historical accuracy

#### Describe alternatives you've considered

historical inaccuracy

#### Testing

spawned it and used it. it works

#### Additional context

sources are : wikipedia, forums and a museum tour

### CODE REVIEW

**Comment by GuardianDll** [comment]
change the Summary to:

```
Balance "Swaps the khopeshes dmg values to reflect irl use, and change the description to reflect it"
```

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -740,14 +740,14 @@
     "type": "TOOL",
     "category": "weapons",
     "name": { "str": "khopesh", "str_pl": "khopeshes" },
-    "description": "This ancient bronze weapon features a curved, sickle-like blade sharpened on the outside edge.  Associated with the New Kingdom period of Ancient Egypt, it was designed mainly to hack through the light armor common to the region.",
+    "description": "This ancient bronze weapon features a curved, sickle-like blunt blade on the outside edge.  Associated with the New Kingdom period of Ancient Egypt, it was designed mainly to disarm enemies with its hook like shape.",
```
Comment (TheShadowFerret) [inline data/json/items/melee/swords_and_blades.json:743]:
```suggestion
    "description": "This ancient bronze weapon features a curved, sickle-like blunt blade on the outside edge.  Associated with the New Kingdom period of Ancient Egypt, it was designed mainly to disarm enemies with its hook-like shape.",
```

**Comment by Amoebka** [comment]
This doesn't seem to be right at all. Wikipedia (a source you listed yourself) says they were sharp on the outside edge ("the blade is only sharpened on the outside portion of the curved end"), not blunt. The inside curve is blunt and used to catch weapons, the outer curve is for slashing. It's a sword, not a baseball bat.
The fully blunt ones are ceremonial weapons never meant for combat, not the "standart issue" variants. That's not what the in-game item represents.

**Comment by fairyarmadillo** [comment]
Yeah it's an axe-sword, these were used to kill people in war. If you're trying to make a fake version of the weapon, that should be a distinct item, but khopeshes are sharp.

**Comment by Aerin-of-the-Toast** [comment]
Also, not to dogpile on you or anything Greg, but even if the khopesh was blunt, you'd have to change the damage by a lot. 27 bash (and 9 extra cut) is *vastly* too high for what would be a blunt 1-handed hook sword thingie, so if you wanted to make a blunt version as a fake variant similar to other blunt swords you'd have to lower the damage a lot.

**Pull request closed without merging**